### DISCUSSION TITLE
3.0.0 打包体积问题。

### DISCUSSION THREAD

**Discussion by shibingli** [imzbf/md-editor-v3#259] [Q&A]
我试着在项目引用md-editor，发现打包后的体积有700K。    `701.59 kB │ gzip: 238.00 kB` 

如果我只用`previewOnly`模式，可以减少包的体积么？

**Comment by imzbf** [comment]
目前没有办法，之前尝试过独立出来，但是有不少问题作者没时间处理。影响很大的话考虑用2的版本替代一下，基础语法上没有太大的区别

**Comment by imzbf** [comment]
这个问题作者会持续尝试优化

**Comment by shibingli** [comment]
期待

**Comment by imzbf** [comment]
4.0.0发布了，按照文档导入`MdPreview`组件，体积会大幅下降

**Reply by shibingli** [reply]
太好了，感谢。
